### PR TITLE
Document how to use Gitlab

### PR DESCRIPTION
**Pull request by garbas**

### CODE REVIEW

**Comment by garbas** [comment]
An example of an successful Gitlab pipeline -> https://gitlab.com/flox-examples/floxdocs/-/jobs/7836482849

File: docs/tutorials/ci-cd.md
Comment (ghudgins) [inline docs/tutorials/ci-cd.md:1]: non-blocking nit: there's a trailing `(r` on the `flox/install-flox-action` line

**Review by bryanhonof — APPROVED** [review]

File: docs/tutorials/ci-cd.md [hunk 1]
```diff
@@ -81,4 +81,19 @@ jobs:
 1. The `install` command will install the latest Flox version. You can change
    the `channel` and `version` option which allows you to select excatly which
    version of Flox to install.
-2. The `activate` command runs a command in the context of a Flox environment.
\ No newline at end of file
+2. The `activate` command runs a command in the context of a Flox environment.
+
+
+## GitLab
+
```
Comment (ghudgins) [inline docs/tutorials/ci-cd.md:88]: could be nice to have a small amount of explainer of what you're going to do in GitLab (even if it's just "similar to GitHub, we are going to setup ci.yaml")

**Review by ghudgins — APPROVED** [review]
comments were non-blocking. LGTM

**Pull request merged**